src/gui/org/deidentifier/arx/gui/model/ModelEvent.java

```diff
@@ -122,7 +122,10 @@
         COST_BENEFIT_MODEL,
         
         /** SELECTED_CLASS_VALUE */
-        SELECTED_CLASS_VALUE
+        SELECTED_CLASS_VALUE,
+        
+        /** STATISTICAL_CLASSIFIER */
+        STATISTICAL_CLASSIFIER
```

Comment:
SELECTED_FEATURES_OR_CLASSES  is being used to update the following views:
(1) Classification accuracy
(2) ROC curves
(3) Classification attributes (features/classes)

If we use the same event when changing the classifier type or configuration parameters, (3) is being updated unnecessarily. Do you wanna use the same event anyway?

FYI: In the current version, I have implemented  the views (1) and (2) to listen to the new event 'STATISTICAL_CLASSIFIER'